app/models/pageflow/draft_entry.rb

```diff
@@ -28,6 +29,11 @@ def initialize(entry, draft = nil)
       @draft = draft || entry.draft
     end
 
+    # So we can always get to the original Entry title.
+    def entry_title
+      entry.title
+    end
+
```

Comment:
It's not delegating `title`, this is a new method `entry_title` which always points at the entry title. I did this so that you can overwrite in the draft, because it's only used for the placeholder.

Do you have ideas to do this differently? The `title` is still delegated, that hasn't changed.